scripts/apiDev.js

```diff
@@ -0,0 +1,126 @@
+const { join } = require('path');
```

Comment:
Do u think error handling here for riskier operations that might fail would be overkill @cdcabrera ?

Reply:
In the related NodeJS scripting... We will say, good error handling is the foundation to quick debugging. 

An explanation to the script
1. You may be referencing the console logging to help call out missing args/params from the script... Overkill we can sort of see in the sense
   - ...that all of this work is a one-time patch where no-one will ever change this script again... 
   - But it may not be overkill if this work is applicable to the v2 branch, where we may modify some of this work.
1. We wrap the the related  `curl` in a try/catch. If that's the case, overkill that's debatable
   - ... yes it could be overkill, but in this case we used it to help relate error color coding in the console output to help sighted users have error logging that stands out from the other logging

If you're referencing something else we'll need more detail

Reply:
Thanks for the comprehensive answer